src/sentry/autofix/webhooks.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+from sentry import analytics
+from sentry.autofix.utils import get_autofix_state_from_pr_id
+from sentry.models.organization import Organization
+from sentry.utils import metrics
+
+AUTOFIX_GITHUB_APP_USER_ID = 157164994
```

Comment:
Should this value come in through settings and not be hard coded?

Reply:
Or some way to make it variable so that users can control the value, while still being locked down for our use case

Reply:
let's move it to settings but we can follow up on configurability later as we haven't set up autofix to run locally yet